WalletWasabi.Gui/ViewModels/AddressViewModel.cs

```diff
@@ -132,7 +132,7 @@ public AddressViewModel(HdPubKey model, KeyManager km)
 				.Merge(DisplayAddressOnHw.ThrownExceptions)
 				.Subscribe(ex =>
 				{
-					NotificationHelpers.Error(ex.ToTypeMessageString());
+					NotificationHelpers.Error("Could not find the device. Make sure it is connected.");
```

Comment:
```suggestion
					NotificationHelpers.Error("Could not find the hardware wallet. Make sure it is connected.");
```

Reply:
I can say the same as before. More things can happen here.